mindsdb/integrations/handlers/openbb_handler/requirements.txt

```diff
@@ -0,0 +1 @@
+openbb==4.0.0a2
```

Comment:
Should we keep this since it looks like a pre-release version?

Reply:
Yes, we will remove the pre-release version once the main package points to v4. But for now this makes it work, whereas v3 would likely break due to amount of dependencies